packages/launcher-component/src/flows/launch-flow.tsx

```diff
@@ -106,7 +106,12 @@ export function LaunchFlow(props: LaunchFlowProps) {
   const progressEvents = run.status === Status.RUNNING && run.result && run.result.events;
   const progressEventsResults = run.status === Status.RUNNING && run.result && run.statusMessages;
 
-  const links = run.statusMessages.filter(m => m.data).map(m => ({[m.statusMessage]: m.data!.location}))!.reduce(
+  const links = run.statusMessages.filter(m => m.data).map(m => {
+    if (m.data!.routes) {
```

Comment:
@edewit can't we receive route and location at the same time?

Reply:
no don't think so, it's either routes or location